modules/dynamic-plugins/proc-enable-plugins-rhdh-container-image.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc-enable-plugins-rhdh-container-image_{context}"]
+= Enabling plugins added in the {product-very-short} container image
+
+In the {product-very-short} container image, a set of dynamic plugins is preloaded to enhance functionality. However, due to mandatory configuration requirements, most of the plugins are disabled.
+
+You can enable and configure the plugins in the {product-very-short} container image, including how to manage the default configuration, set necessary environment variables, and ensure the proper functionality of the plugins within your application.
+
+.Prerequisites
+* You have access to the link:https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins.default.yaml[`dynamic-plugins.default.yaml`] file, which lists all preloaded plugins.
+* You have deployed the {product-very-short} application, and have access to the logs of the `install-dynamic-plugins` init container.
+* You have the necessary permissions to modify plugin configurations and access the application environment.
+* You have identified and set the required environment variables referenced by the plugin's default configuration. These environment variables must be defined in the Helm Chart or Operator configuration.
+
+.Procedure
+. Locate the link:https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins.default.yaml[`dynamic-plugins.default.yaml`] file in your setup.
+. Start your {product-very-short} application and access the logs of the `install-dynamic-plugins` init container within the {product-very-short} pod.
+. Identify the link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/dynamic_plugins_reference/index#red-hat-supported-plugins[Red Hat supported plugins] that are disabled by default.
+. Open the plugin configuration file and locate the plugin entry you want to activate.
+. Modify the `disabled` field to `false` and specify the package name as follows:
++
+--
+.Example plugin configuration
+[source,yaml]
+----
+plugins:
+  - disabled: false
+    package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic
+----
+--
+. Optional: Replace the default plugin configuration from the `dynamic-plugins.default.yaml` file by adding a `pluginConfig` entry to the plugin configuration file.
```

Comment:
users should never touch dynamic-plugins.default.yaml, they should provide their configuration as we have it described in https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.4/html/installing_and_viewing_plugins_in_red_hat_developer_hub/rhdh-installing-rhdh-plugins_title-plugins-rhdh-about#rhdh-installing-rhdh-plugins_title-plugins-rhdh-about